backend/src/main/java/org/sefglobal/core/service/CertificateService.java

```diff
@@ -0,0 +1,59 @@
+package org.sefglobal.core.service;
+
+import org.sefglobal.core.exception.ResourceNotFoundException;
+import org.sefglobal.core.model.Certificate;
+import org.sefglobal.core.repository.CertificateRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CertificateService {
+
+    private final CertificateRepository certificateRepository;
+
+    public CertificateService(CertificateRepository certificateRepository) {
+        this.certificateRepository = certificateRepository;
+    }
+
+    /**
+     * Retrieves all the certificates in the repository
+     * @return List of certificates objects
+     */
+    public List<Certificate> getCertificates(){
+        return certificateRepository.findAll();
```

Comment:
Shouldn't this method take `offset` and `limit` as params and return a page. Unless this will query the whole table. WDYT @jayasanka-sack

Reply:
Mmm.. yeah this should be paginated

Reply:
Yeah, that's fine @Gravewalker666 can you add an issue for this and close the review

Reply:
https://github.com/sef-global/sef-core/issues/33

Reply:
I created an issue to resolve this.